app/src/geo/google.js

```diff
@@ -27,10 +26,43 @@ module.exports = {
     },
 
     getAddress: function(location, callback) {
-        geocoder.reverse(location, function(err, res) {
+        geocoder.reverse(location, function(err, data) {
```

Comment:
why rename res to data here 🤔

Reply:
data is the raw data from geocoder, while res is what is returned to the callback of getAddress. 

It was an attempt to clear up the dataflow in my own head.